network/p2p/inspector/validation/control_message_validation_inspector.go

```diff
@@ -599,3 +698,21 @@ func (c *ControlMsgValidationInspector) checkClusterPrefixHardThreshold(nodeID f
 	}
 	return gauge <= c.config.ClusterPrefixHardThreshold
 }
+
+// logAndDistributeErr logs the provided error and attempts to disseminate an invalid control message validation notification for the error.
+func (c *ControlMsgValidationInspector) logAndDistributeAsyncInspectErr(req *InspectMsgRequest, count uint64, err error) {
+	lg := c.logger.With().
+		Bool(logging.KeySuspicious, true).
+		Str("peer_id", req.Peer.String()).
+		Str("ctrl_msg_type", string(req.validationConfig.ControlMsg)).
+		Uint64("ctrl_msg_count", count).Logger()
+
+	lg.Error().Err(err).Msg("rpc control message async inspection failed")
+
+	err = c.distributor.Distribute(p2p.NewInvalidControlMessageNotification(req.Peer, req.validationConfig.ControlMsg, count, err))
+	if err != nil {
+		lg.Error().
+			Err(err).
+			Msg("failed to distribute invalid control message notification")
```

Comment:
There are logs in this file all having the `Msg("failed to distribute invalid control message notification")`, please add a `Bool(logging.KeyNetworkingSecurity, true).` to all of them.

Reply:
https://github.com/onflow/flow-go/pull/4574/commits/fb521fcda685b9bc39c8e67a9561de712f26b178